.github/workflows/release.yml

```diff
@@ -43,3 +47,19 @@ jobs:
         with:
           github_token: ${{ secrets.SEMANTIC_RELEASE_PAT }}
           branch: ${{ github.ref }}
+
+  release-cocoapod:
+    name: Release CocoaPod
+    runs-on: macOS-latest
+    needs:
+      - tag-repo
+
+    if: ${{ needs.tag-repo.outputs.releaseType != '' }}
```

Comment:
👀 hopefully the `needs` context is set here when it goes to check this condition. will need to keep an eye on this

Reply:
Looks like this output never gets set if a new tag isn't made in `github-tag-action`. Do we need to account for `releaseType: undefined`?

Reply:
I've had this check other places and haven't had an issue with `chore` commits. so I think we're good... surprisingly

Reply:
my guess is that GH is defaulting that output to `''`